languoids/tree/grea1241/midd1355/midd1323/juwo1236/akak1253/md.ini

```diff
@@ -30,7 +30,8 @@ lexvo =
 multitree = aky
 
 [classification]
-sub = **hh:hv:Manoharan:Andamanese**
+sub = **hh:hv:Manoharan:Andamanese** **hh:typ:ComrieZamponi:Great-Andamanese**
 subrefs = 
-	**hh:hv:Manoharan:Andamanese**
+	hh:hv:Manoharan:Andamanese
+	hh:typ:ComrieZamponi:Great-Andamanese
```

Comment:
When I unified the way sources are referenced from INI files, I adopted the convention to **always** mark source keys with `**<key>**`. Btw. this is something that `glottolog check tree` should catch.